cmd/validate/image.go

```diff
@@ -226,14 +231,59 @@ func validateImageCmd(validate imageValidationFunc) *cobra.Command {
 
 			appComponents := data.spec.Components
 
+			var wg sync.WaitGroup
+			errChan := make(chan error, len(data.policy.Spec().Sources))
+			evaluatorsChan := make(chan *evaluator.Evaluator, len(data.policy.Spec().Sources))
+			for _, sourceGroup := range data.policy.Spec().Sources {
+				wg.Add(1)
+				go func(sourceGroup v1alpha1.Source) {
```

Comment:
Likely to have a less impact here, but blindly starting an unbounded number of goroutines is problematic. See the worker/jobs pattern used below for an alternative.

Also, there's probably not a whole lot of value in fetching multiple source groups in parallel.